pkgs/build-support/build-fhs-userenv-bubblewrap/default.nix

```
@@ -71,8 +71,8 @@ let
       "ca-certificates"
       "pki"
     ];
-  in concatStringsSep "\n  "
-  (map (file: "--ro-bind-try $(${coreutils}/bin/readlink -m /etc/${file}) /etc/${file}") files);
+  in concatStringsSep " "
+    (map (file: "\"/etc/${file}\"") files);
```

If you apply the escapeShellArgs suggestion...
```suggestion
  in map (path: "/etc/${path}") files;
```
(changed `file` to `path` since some of them aren't regular files at all)